src/ouroboros_gt/gt_place_recognition.py

```diff
@@ -1,17 +1,17 @@
-import numpy as np
+from __future__ import annotations
```

Comment:
Is this necessary? Might have missed usage of a newer typing feature but annotations should be fine in 3.8 (I just tend to mix direct annotations with docstrings because I want newer typing features like Self or specifying numpy array sizes)